contracts/data/WitnetBoardDataACLs.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "./WitnetBoardData.sol";
+
+/**
+ * @title Witnet Access Control Lists storage layout, for centralized request boards.
+ * @author Witnet Foundation
+ */
+abstract contract WitnetBoardDataACLs is WitnetBoardData {  
+  struct SWitnetBoardACLs {
+    mapping (address => bool) isReporter_;
+  }
+
+  constructor() {
```

Comment:
maybe we could include in the constructor inputs the list of addresses that can be reporters, at the moment only the sender will be a reporter.  If there is going to be only one reporter with the onlyowner modifier should be enough

Reply:
The owner/operator can call `setReporters(address[])` at anytime. Besides, a list of reportes can be specified whenever a delegated WitnetRequestBoard instance is initialized (e.g. when upgrading and older implementation).